### DISCUSSION TITLE
posts/%E5%81%A5%E8%BA%AB%E6%97%A5%E8%AE%B0/2023-10-11/

### DISCUSSION THREAD

**Discussion by giscus** [dstweihao/dst-hugo-giscus#6] [Announcements]
# posts/%E5%81%A5%E8%BA%AB%E6%97%A5%E8%AE%B0/2023-10-11/

无氧 无
有氧 

http://localhost:1313/posts/%E5%81%A5%E8%BA%AB%E6%97%A5%E8%AE%B0/2023-10-11/

**Comment by dstweihao** [comment]
验证一下
